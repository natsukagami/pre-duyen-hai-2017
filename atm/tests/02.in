17 1
-0737021028846414
