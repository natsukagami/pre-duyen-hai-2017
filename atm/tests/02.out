0737021028846414
